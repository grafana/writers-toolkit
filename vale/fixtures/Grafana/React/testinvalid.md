Don't use:

- ReactJS
- reactJS
- react.js
- react.JS
- react JS
