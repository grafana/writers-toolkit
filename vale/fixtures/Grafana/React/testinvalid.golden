testinvalid.md:3:3:Grafana.React:Use 'React' instead of 'ReactJS'.
testinvalid.md:4:3:Grafana.React:Use 'React' instead of 'reactJS'.
testinvalid.md:5:3:Grafana.React:Use 'React' instead of 'react.js'.
testinvalid.md:6:3:Grafana.React:Use 'React' instead of 'react.JS'.
testinvalid.md:7:3:Grafana.React:Use 'React' instead of 'react JS'.
